### PR TITLE
Building google searchbox script

### PR DESCRIPTION
**Pull request by ccamargo2294**
#### What is the purpose of this pull request?
Adding Google Search Box script in the head to show the input search for the store in google search

image

#### How should this be manually tested?
go to https://vtexcamilo--exitocol.myvtex.com/ in the elements should be with the script

#### Screenshots or example usage

image

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
   },
-  "mustUpdateAt": "2019-04-02",
+  "mustUpdateAt": "2019-08-29",
   "scripts": {
     "postreleasy": "vtex publish --verbose"
   },
   "dependencies": {
-    "vtex.styleguide": "9.x"
-  }
+    "vtex.styleguide": "9.x",
```
Comment (jgfidelis) [inline manifest.json]: none of these dependencies are needed

**Comment by jgfidelis** [comment]
shouldn't  this be a pixel, use the pixel builder, etc?

File: store/interfaces.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "base-store-component": {
-    "component": "Example"
+  "google-search-box": {
```
Comment (klzns) [inline store/interfaces.json]: No need to create an interface right now. You can remove this file

File: react/components/SearchBox.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+const GoogleSearchBox = () => {
+  const hostname = canUseDOM ? window.location.hostname : global.__hostname__
+  const protocol = canUseDOM ? window.location.protocol : global.__protocol__
+  return (
+    <Helmet>
+      <script type="application/ld+json">
+        {`
+            "@context": "http://schema.org",
+            "@type": "WebSite",
+            "url": \`${protocol}//${hostname}/\`,
```
Comment (klzns) [inline react/components/SearchBox.tsx]: Ops, add 
```suggestion
            "url": \`${protocol}//${hostname}/{__RUNTIME__.rootPath || ''}\`,
```

Make sure this doesnt break SSR

**Pull request merged**